src/color/setting.js

```diff
@@ -615,6 +611,7 @@ p5.prototype.noStroke = function() {
  * strokeWeight(4);
  * stroke(51);
  * rect(20, 20, 60, 60);
+ * describe('60×60 white rect at center. Dark charcoal grey outline.');
```

Comment:
to make this more readable could you change it to 'white rect at center with dark charcoal grey outline'

Reply:
it'd be great if you could do this to the following descriptions too @Zearin

Reply:
Okie dokie...

Reply:
Done in 05ba1265.